client/ayon_blender/addon.py

```diff
@@ -34,7 +34,10 @@ def add_implementation_envs(self, env, _app):
         # If `AYON_BLENDER_USE_SYSTEM_PATH` is set use `BLENDER_SYSTEM_PATH`
         # to initialize the Blender startup environment, otherwise use the
         # `BLENDER_USER_PATH`.
-        use_system_path = env_value_to_bool("AYON_BLENDER_USE_SYSTEM_PATH")
+        use_system_path = env_value_to_bool(
+            env_key="AYON_BLENDER_USE_SYSTEM_PATH",
+            value=env.get("AYON_BLENDER_USE_SYSTEM_PATH"),
+        )
```

Comment:
I think the `env_key` is redundant here if `value` is passed. Also, a comment may be nice here.
```suggestion
        # The launch env is not applied yet, so we pass the env value
        # from the launch environment
        use_system_path = env_value_to_bool(
            value=env.get("AYON_BLENDER_USE_SYSTEM_PATH"),
        )
```